ocs_ci/ocs/ui/deployment_ui.py

```diff
@@ -75,9 +75,11 @@ def install_ocs_operator(self):
         if self.operator_name is ODF_OPERATOR:
             try:
                 self.navigate_installed_operators_page()
-                self.do_click(locator=self.dep_loc["refresh_popup"], timeout=300)
+                self.do_click(locator=self.dep_loc["refresh_popup"], timeout=500)
             except Exception as e:
+                self.take_screenshot()
```

Comment:
What do you think about use in such cases our self-written ElementNotFOundException(selenium.common.exceptions.NoSuchElementException) and take_screenshot() every time when we get it? So we can implicitly take screenshot in every such occasion + remove from 21 calls within frame.
@OdedViner ?

Reply:
If it's fixing the issue I am for merge this and open the issue to improve the code as Daniel suggested.

Reply:
I've opened issue, assigned to myself
@OdedViner WDYT about idea itself? Concerns?

Reply:
hi @DanielOsypenko , we take screenshots when we get exception and and in close function.

https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/ui/base_ui.py#L109
https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/ui/base_ui.py#L144
https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/ui/base_ui.py#L1166